### ISSUE TITLE
Event trigger

### ISSUE THREAD

**Issue by neomicho85** [Via-profit/js-form-validator#7]
As i read in the doc the validation can be trigger
input change, blur, input...
form submit
But there is another way to validate the form, like a button (not submit type)?

**Comment by Via-profit** [comment]
yes, you can use API (validate method). But it`s not recommended, because web browsers expects that in form there should be an <input type="submit">, and when user hit the 'Enter' on keyboard, browser should trigger onSubmit event. If form doesn`t contains <input type="submit">, this event will not be triggered :(
